package.json

```diff
@@ -28,6 +28,7 @@
         "post-build": "npm run copy-theme-sheet && npm run copy-package && npm run copy-readme && npm run pack",
         "prettier:check": "prettier --check src/**/*.{js,ts,tsx,scss}",
         "prettier:fix": "prettier --write src/**/*.{js,ts,tsx,scss}",
+        "start": "npm run storybook",
```

Comment:
nice QOL improvement

Reply:
yeah. just seems like it aligns with stadards. You still have to run `npm run start` but at least you don't have to know we use storybook.